lamindb_setup/dev/upath.py

```diff
@@ -168,6 +168,15 @@ def upload_from(self, path, print_progress: bool = False, **kwargs):
 def synchronize(self, filepath: Path, **kwargs):
     """Sync to a local destination path."""
     if not self.exists():
+        warn_str = f"The original path {self} does not exist anymore"
+        if filepath.exists():
+            warn_str += (
+                f". However, the local path {filepath} still exists, you might want to"
```

Comment:
I would add a new line here, so that it's easy to read. Forget about the `.`

Reply:
done